### DISCUSSION TITLE
Why is Tesla endpoint https://owner-api.teslamotors.com /oauth/token not working?

### DISCUSSION THREAD

**Discussion by HossamGad** [timdorr/tesla-api#326] [Q&A]
I'm trying to get token by this endpoint https://owner-api.teslamotors.com/oauth/token in postman but it didn't work.I always get Bad request 400. Anyone knows how to solve this problem? 

Here is response body:

{
  "response": {},
  "error": "Endpoint deprecated: Please update your App.",
  "error_description": ""
}

Thanks

**Comment by timdorr** [comment]
See #260
